x-pack/plugins/cloud_security_posture/server/create_indices/latest_findings_mapping.ts

```diff
@@ -49,7 +48,25 @@ export const latestFindingsMapping: MappingTypeMapping = {
     },
     resource: {
       properties: {
-        filename: {
+        type: {
+          type: 'text',
```

Comment:
Hey, just a thought. If we are introducing new fields worth discussing if we can shift to ECS convention for text fields https://www.elastic.co/guide/en/ecs/8.2/ecs-conventions.html#_ecs_convention_for_indexing_text_fields

Reply:
Sounds good, I'll update the mapping according to the ECS convention.